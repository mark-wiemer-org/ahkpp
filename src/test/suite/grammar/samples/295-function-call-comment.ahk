; [Issue #295](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/295)
Foo();NotComment
Foo() ;Comment
