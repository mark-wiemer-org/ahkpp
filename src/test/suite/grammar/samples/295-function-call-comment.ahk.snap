>; [Issue #295](https://github.com/mark-wiemer/vscode-autohotkey-plus-plus/issues/295)
#^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ source.ahk comment.line.semicolon.ahk
>Foo();NotComment
#^^^ source.ahk entity.name.function.ahk
#   ^ source.ahk punctuation.bracket.ahk
#    ^ source.ahk punctuation.bracket.ahk
#     ^ source.ahk
#      ^^^^^^^^^^ source.ahk variable.def.ahk
>Foo() ;Comment
#^^^ source.ahk entity.name.function.ahk
#   ^ source.ahk punctuation.bracket.ahk
#    ^ source.ahk punctuation.bracket.ahk
#     ^ source.ahk
#      ^ source.ahk comment.line.semicolon.ahk punctuation.definition.comment.ahk
#       ^^^^^^^ source.ahk comment.line.semicolon.ahk
>